twpca/model.py

```diff
@@ -315,37 +320,18 @@ def fit(self, optimizer=None, niter=1000, lr=1e-3, progressbar=True):
             lr (optional): float, learning rate for the optimizer (default: 1e-3)
             progressbar (optional): whether to print a progressbar (default: True)
         """
-
```

Comment:
ya exactly. seems more straightforward than passing in lists of learning rates.

on a related note, can we just get rid of the `optimizer` argument to `fit`? How common is it that you would want to restart the optimizer?